src/HealthcareApis/HealthcareApis/Az.HealthcareApis.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.2'
+ModuleVersion = '1.0.0'
```

Comment:
Being at 1.0 means that your service is GA and you are opting in to the breakign change policy (only 2 breaking changes per year, during designated breakign change releases).

Reply:
Yes. We are Ga'ing this servce. I will update the same in the PR description.

Reply:
Please revert changes to thsi file, we need to make this change (and change versions in a couple of other places as well) as part of the release.  We are tracking your version update for this release [here](https://github.com/Azure/azure-powershell/issues/10169)